shell/platform/windows/flutter_window_winuwp.h

```diff
@@ -0,0 +1,189 @@
+      winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  // Notifies current |WindowBindingHandlerDelegate| of key up events.
+  void OnKeyUp(winrt::Windows::Foundation::IInspectable const&,
+               winrt::Windows::UI::Core::KeyEventArgs const& args);
+
+  // Notifies current |WindowBindingHandlerDelegate| of key down events.
+  void OnKeyDown(winrt::Windows::Foundation::IInspectable const&,
+                 winrt::Windows::UI::Core::KeyEventArgs const& args);
+
+  // Notifies current |WindowBindingHandlerDelegate| of character received
+  // events.
+  void OnCharacterReceived(
+      winrt::Windows::Foundation::IInspectable const&,
+      winrt::Windows::UI::Core::CharacterReceivedEventArgs const& args);
+
+  // Creates a visual representing the emulated cursor and add to the  visual
+  // tree
+  winrt::Windows::UI::Composition::Visual CreateCursorVisual();
+
+  // Test is current context is running on an xbox device and perform device
+  // specific initialization.
+  void ConfigureXboxSpecific();
+
+  // Converts from logical point to physical X value.
+  double GetPosX(winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  // Converts from logical point to physical Y value.
+  double GetPosY(winrt::Windows::UI::Core::PointerEventArgs const& args);
+
+  // Token representing current worker thread.
+  winrt::Windows::Foundation::IAsyncAction worker_loop_{nullptr};
+
+  // Backing CoreWindow. nullptr if not set.
+  winrt::Windows::UI::Core::CoreWindow window_{nullptr};
+
+  // Pointer to a FlutterWindowsView that can be
+  // used to update engine windowing and input state.
+  WindowBindingHandlerDelegate* binding_handler_delegate_;
+
+  // Current active compositor. nullptr if not set.
+  winrt::Windows::UI::Composition::Compositor compositor_{nullptr};
+
+  // Current CompositionTarget for binding the
+  // rendering context to the CoreWindow. nullptr if not set.
+  winrt::Windows::UI::Composition::CompositionTarget target_{nullptr};
+
+  // Composition tree root object.
+  winrt::Windows::UI::Composition::ContainerVisual visual_tree_root_{nullptr};
+
+  // Composition visual representing the emulated
+  // cursor visual.
+  winrt::Windows::UI::Composition::Visual cursor_visual_{nullptr};
+
+  // Compositor object that represents the render target binding the backing
+  // SwapChain to the CoreWindow.
+  winrt::Windows::UI::Composition::SpriteVisual render_target_{nullptr};
+
+  // Is current context is executing on an XBOX
+  // device.
+  bool running_on_xbox_ = false;
+
+  // Current X overscan compensation factor.
+  float xbox_overscan_x_offset_ = 0.0f;
+
+  // Current Y overscan compensation factor.
+  float xbox_overscan_y_offset_ = 0.0f;
+
+  // Most recent display information.
+  winrt::Windows::Graphics::Display::DisplayInformation current_display_info_{
+      nullptr};
+
+  // Multipler used to map controller velocity to an appropriate scroll input.
+  const double kControllerScrollMultiplier = 3;
```

Comment:
Should this just be a file-scoped `static constexpr` in the .cc?

Reply:
Yup, fixed in next commit